familias/forms.py

```diff
@@ -0,0 +1,57 @@
+from django.forms import ModelForm, ChoiceField
+from .models import Familia, Integrante, Alumno, Tutor
+
+
+class FamiliaForm(ModelForm):
+    class Meta:
+        model = Familia
+        fields = ('numero_hijos_diferentes_papas',
+                  'estado_civil',
+                  'localidad')
+
+    def __init__(self, *args, **kwargs):
+        super(FamiliaForm, self).__init__(*args, **kwargs)
+        for field_name, field in self.fields.items():
+            field.widget.attrs['class'] = 'form-control'
+
+
+class IntegranteForm(ModelForm):
+    OPCION_ROL_NINGUNO = 'Ninguno'
+    OPCION_ROL_TUTOR = 'Tutor'
+    OPCION_ROL_ALUMNO = 'Alumno'
+    OPCIONES_ROL = ((OPCION_ROL_NINGUNO, 'Ninguno'),
+                    (OPCION_ROL_TUTOR, 'Tutor'),
+                    (OPCION_ROL_ALUMNO, 'Alumno'))
+
+    Rol = ChoiceField(choices=OPCIONES_ROL, required=False)
```

Comment:
Also i'm pretty sure there's a way to add the date time picker widget for fecha de nacimiento. right now you have to enter it manually